include/deal.II/particles/generators.h

```diff
@@ -207,7 +210,9 @@ namespace Particles
                        ParticleHandler<dim, spacedim> &particle_handler,
                        const Mapping<dim, spacedim> &  mapping =
                          StaticMappingQ1<dim, spacedim>::mapping,
-                       const ComponentMask &components = ComponentMask());
+                       const ComponentMask &components = ComponentMask(),
+                       const std::vector<std::vector<double>> &properties =
+                         std::vector<std::vector<double>>());
```

Comment:
```suggestion
                         {});
```
as done in `insert_global_particles`.